### ISSUE TITLE
Hello! When I click on the link to open the interface, the interface does not open, I get the message "504 Gateway Time-out" What will be the problem? Thanks in advance!

### ISSUE THREAD

**Issue by eduardodp-pixel** [TheLastBen/fast-stable-diffusion#1494]

**Comment by TheLastBen** [comment]
If the error persists, check the box "use localtunnel"

**Comment by AdamKnapp1** [comment]
Hello! I'm having the same problem, i think something it's going on with gradio servers. If you find a solution, please tell me  :).

**Comment by creativechimp** [comment]
First time user and I just got this issue.  Clicked use localtunnel and it fixed the issue

**Comment by AdamKnapp1** [comment]
> First time user and I just got this issue. Clicked use localtunnel and it fixed the issue

I clicked on localtunnel and it works, but for some reason i can't generate good quality images (1024x1024), like, i just click generate and nothing happens.

**Comment by creativechimp** [comment]
I haven't tried that yet

**Comment by AdamKnapp1** [comment]
Tell me if you can then, i can't even generate images, i think the error it's something related to gradio servers.

**Comment by creativechimp** [comment]
Not sure if you're supposed to nut I connected colab to github in settings

**Comment by cosmicoxytocin** [comment]
> Tell me if you can then, i can't even generate images, i think the error it's something related to gradio servers.

I think your assumption is correct on it being an issue with Gradio servers. I've been able to use the localtunnel after a few relaunches.

**Comment by AdamKnapp1** [comment]
> > Tell me if you can then, i can't even generate images, i think the error it's something related to gradio servers.
> 
> I think your assumption is correct on it being an issue with Gradio servers. I've been able to use the localtunnel after a few relaunches.

Yup, i think that's definitely the problem, localtunnel for me stills not working, but may be after a few relaunches it works. I think the best option now it's just wait until gradio servers works again.

**Comment by navarisun** [comment]
Is there any solution for this ? The whole day i try to access the generated gradio or tunnel login with no hope.
